app/models/subscriber.rb

```diff
@@ -0,0 +1,32 @@
+class Subscriber
```

Comment:
What are `models`? Are they just classes that connect to the db? Or all of our business objects?

Also, what are `services`? What makes them different than a regular object?